velox/substrait/VeloxToSubstraitPlan.cpp

```diff
@@ -35,12 +35,28 @@ ::substrait::AggregationPhase toAggregationPhase(
       return ::substrait::AGGREGATION_PHASE_INTERMEDIATE_TO_RESULT;
     }
     default:
-      VELOX_NYI(
+      VELOX_UNSUPPORTED(
           "Unsupported Aggregate Step '{}' in Substrait ",
           mapAggregationStepToName(step));
   }
 }
 
+::substrait::SortField_SortDirection toSortDirection(
+    core::SortOrder sortOrder) {
+  if (sortOrder.isNullsFirst() && sortOrder.isAscending()) {
+    return ::substrait::SortField_SortDirection_SORT_DIRECTION_ASC_NULLS_FIRST;
+  } else if (sortOrder.isNullsFirst() && !sortOrder.isAscending()) {
```

Comment:
else after return is redundant; please, remove.

Reply:
fix.